docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -0,0 +1,294 @@
+
+
+## Optional: Choose SinkBinding namespace selection behavior
+
+The SinkBinding object operates in one of two modes: `exclusion` or `inclusion`.
+
+The default mode is `exclusion`.
+In exclusion mode, SinkBinding behavior is enabled for the namespace by default.
+To disallow a namespace from being evaluated for mutation you must exclude it
+using the label `bindings.knative.dev/exclude: true`.
+
+In inclusion mode, SinkBinding behavior is not enabled for the namespace.
+Before a namespace can be evaluated for mutation, you must
+explicitly include it using the label `bindings.knative.dev/include: true`.
+
+To set the SinkBinding object to inclusion mode:
+
+1. Change the value of `SINK_BINDING_SELECTION_MODE` from `exclusion` to `inclusion` by running:
+
+    ```bash
+    kubectl -n knative-eventing set env deployments eventing-webhook --containers="eventing-webhook" SINK_BINDING_SELECTION_MODE=inclusion
+    ```
+
+2. To verify that `SINK_BINDING_SELECTION_MODE` is set as desired, run:
+
+    ```bash
+    kubectl -n knative-eventing set env deployments eventing-webhook --containers="eventing-webhook" --list | grep SINK_BINDING
+    ```
+
+
+## Create a namespace
+
+If you do not have an existing namespace, create a namespace for the SinkBinding:
+
+```bash
+kubectl create namespace <namespace>
+```
+Where `<namespace>` is the namespace that you want your SinkBinding to use.
+For example, `sinkbinding-example`.
+
+!!! note
+    If you have selected inclusion mode, you must add the
+    `bindings.knative.dev/include: true` label to the namespace to enable
+    SinkBinding behavior.
+
+
+## Create a sink
+
+The sink can be any addressable Kubernetes object that can receive events.
+
+If you do not have an existing sink that you want to connect to the SinkBinding,
+create a Knative service.
+
+!!! note
+    To create a Knative service you must have Knative Serving installed on your cluster.
+
+=== "kn"
+
+    Create a Knative service by running:
+
+    ```bash
+    kn service create <app-name> --image <image-url>
+    ```
+    Where:
+    - `<app-name>` is the name of the application.
+    - `<image-url>` is the URL of the image container.
+
+    For example:
+
+    ```bash
+    $ kn service create hello --image gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
+    ```
+
+=== "YAML"
+    1. Create a Knative service by running:
+
+        ```yaml
+        kubectl apply -f - <<EOF
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        metadata:
+          name: <app-name>
+        spec:
+          template:
+            spec:
+              containers:
+                - image: <image-url>
+        EOF
+        ```
+        Where:
+        - `<app-name>` is the name of the application. For example, `event-display`.
+        - `<image-url>` is the URL of the image container.
+        For example, `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display`
+
+
```

Comment:
@omerbensaadon this is an example of where we need content reuse. As I mentioned in [Slack](https://knative.slack.com/archives/C9CV04DNJ/p1622065546110800) there are waaaaaay too many duplicated docs sections explaining the basics of creating a basic service. We should have one good reusable example that we turn into a snippet.